api/cloud/oc_cloud_store.c

```diff
@@ -32,6 +32,18 @@ check oc_config.h and make sure OC_STORAGE is defined if OC_CLOUD is defined.
 #endif
 
 #define CLOUD_STORE_NAME "cloud"
+#define ci_server ci_server
```

Comment:
Capital letters? That's usually the convention for macros.

Reply:
Fixed